src/Backend/Core/Js/ckeditor/samples/old/docprops/docprops.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<!--
+Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.

```

Comment:
can you remove the samples stuff? it is kinda useless

Reply:
Samples removed. See: 23fcd798